src/index.scss

```
@@ -0,0 +1,66 @@
+ 
+// Taken from @blueprintjs/core/lib/css/blueprint.css 
+@charset "UTF-8";
+
+html {
+  -webkit-box-sizing:border-box;
+          box-sizing:border-box;
+}
+
+*, *::before, *::after {
+  -webkit-box-sizing:inherit;
+          box-sizing:inherit;
```

```suggestion
  box-sizing:inherit;
```
I think we can drop the prefixed version. 1. All browsers we care about support `box-sizing` and 2. we're using `autoprefixer` which should add prefixes where necessary.